pontoon/actionlog/utils.py

```diff
@@ -0,0 +1,34 @@
+from pontoon.actionlog.models import ActionLog
+
+
+def log_action(
+    action,
+    user,
+    translation=None,
+    entity=None,
+    locale=None,
+):
+    """Save a new action in the database.
+
+    :arg string action:
+        The type of action that was performed.
+        See models.ActionLog.ACTIONS_TYPES for choices.
+    :arg User user: The User who performed the action.
+    :arg Translation translation: The Translation the action was performed on.
+    :arg Entity entity:
+        The Entity the action was performed on.
+        Only used for the "translation:deleted" action.
+    :arg Locale locale:
+        The Locale the action was performed on.
+        Only used for the "translation:deleted" action.
+
+    :returns: None
+    """
+    action = ActionLog(
+        action_type=action,
+        performed_by=user,
+        translation=translation,
+        entity=entity,
+        locale=locale,
+    )
+    action.save()
```

Comment:
Do we really need this util function, given how simple it is?

Reply:
It's simple but this feature will be used in a lot of places, and I find it easier to import and use this function than import the model and remember to use `.save()` every time. I think it's also more maintainable over time, especially since we know that this feature is going to grow (to log more actions, to be used in more places, etc. ).